app/controllers/content_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ContentController < ApplicationController
+  def accessibility
+    render_content_page :accessibility
+  end
+
+private
+
+  def render_content_page(page_name)
+    markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML.new)
+
+    @converted_markdown = markdown.render(File.read("app/views/content/#{page_name}.md")).html_safe
```

Comment:
Should we sanitize `page_name`?

Reply:
It's set by us in the controller, so I don't think so?

Reply:
:+1: Pure paranoia at this stage